.gitpod.yml

```diff
@@ -1,24 +1,29 @@
 tasks:
-  - name: Deploy Jekyll Docs
-    command: export JEKYLL_ENV=production; cd /workspace/AaC/docs; bundle install && bundle exec jekyll s
+  - name: Serve Sphinx Docs
+    init: |
+      cd /workspace/AaC/docs/
+      sleep 120 # Gitpod often takes a long time to install the python environment
+      source /workspace/AaC/python/venv/bin/activate
+      sphinx-build source gen_docs # Pre-build the docs so that when Gitpod opens the port it's ready to serve
+    command: sphinx-autobuild source gen_docs
 
   - name: VSCode Extension Environment
     init: |
-      cd vscode_extension/
+      cd /workspace/AaC/vscode_extension/
       yarn install
+      sleep 120 # Gitpod often takes a long time to install the python environment
```

Comment:
Get this move around, grouping pertinent things together. Good catch for some clean up.